### PR TITLE
Sync with problem-specs

### PR DESCRIPTION
**Pull request by glennj**
configlet sync --docs
configlet sync --tests

### CODE REVIEW

File: exercises/practice/bottle-song/.meta/tests.toml [hunk 1]
```diff
@@ -9,26 +9,23 @@
...
-[102cbca0-b197-40fd-b548-e99609b06428]
+[f61f3c97-131f-459e-b40a-7428f3ed99d9]
 description = "verse -> single verse -> verse with 2 bottles"
 
-[b8ef9fce-960e-4d85-a0c9-980a04ec1972]
+[05eadba9-5dbd-401e-a7e8-d17cc9baa8e0]
 description = "verse -> single verse -> verse with 1 bottle"
 
-[c59d4076-f671-4ee3-baaa-d4966801f90d]
-description = "verse -> single verse -> verse with 0 bottles"
```
Comment (IsaacG) [inline exercises/practice/bottle-song/.meta/tests.toml:25]: Should this match a unit test change?

Comment (glennj) [reply exercises/practice/bottle-song/.meta/tests.toml:25]: I had a quick look at the history of the canonical data and didn't see that test at all. I might have missed it but I don't know why all the uuids changed.

Comment (glennj) [reply exercises/practice/bottle-song/.meta/tests.toml:25]: Also, the test suite does not contain this test: https://github.com/exercism/awk/blob/main/exercises/practice/bottle-song/test-bottle-song.bats

Comment (glennj) [reply exercises/practice/bottle-song/.meta/tests.toml:25]: @rabestro do you recall how tests.toml was generated when you created bottle-song? 

The appropriate ways are: `bin/configlet create --practice-exercise $exercise` or `bin/configlet sync --tests -e $exercise -u`

Comment (rabestro) [reply exercises/practice/bottle-song/.meta/tests.toml:25]: @glennj I found in my log this commands:
```
./bin/configlet lint
./bin/configlet fmt -e bottle-song
```

Comment (rabestro) [reply exercises/practice/bottle-song/.meta/tests.toml:25]: @glennj The script for test generation:

https://github.com/rabestro/exercism-solutions-awk/blob/main/bowling/prompt.txt

Currently I have no wrapper in PowerShell or bash for the prompt, so I just use it directly in AI system.

**Pull request merged**